### DISCUSSION TITLE
Is there support for Microsoft generic host?

### DISCUSSION THREAD

**Discussion by RollsChris** [spectreconsole/spectre.console#566] [Q&A] [answered]
HI.. 

I was just wondering if there is going to be any support to use Spectre as the logging provider for Generic hots, so adding some extension methods and stuff and some configuration options?

Thanks

**Comment by patriksvensson** [comment, marked as answer]
@RedEyedDog I found this a week ago, which looks promising: https://github.com/verticalsoftware/vertical-spectreconsolelogger
I also know that @agc93 did a stellar job with https://github.com/agc93/spectre.console.extensions.logging, but not sure if it's still compatible with the latest version.

**Reply by RollsChris** [reply]
Thanks ill take a look :)
